resources/secrets/src/lib.rs

```diff
@@ -1,9 +1,35 @@
+//! [Shuttle Secrets](https://docs.shuttle.rs/resources/shuttle-secrets)
+//! A secrets plugin for Shuttle for storing environment secrets.
+//!
+//! # Example
+//!
+//! ```rust,ignore
+//! #[shuttle_runtime::main]
+//! async fn rocket(
+//!     #[shuttle_secrets::Secrets] secret_store: SecretStore,
+//! ) -> ShuttleRocket {
+//!     // get secret defined in `Secrets.toml` file.
+//!     let secret = if let Some(secret) = secret_store.get("MY_API_KEY") {
+//!         secret
+//!     } else {
+//!         return Err(anyhow!("secret was not found").into());
+//!     };
+//!
+//!     let state = MyState { secret };
+//!     let rocket = rocket::build().mount("/", routes![secret]).manage(state);
+//!
+//!     Ok(rocket.into())
+//! }
+//!
+//! ```
+
```

Comment:
It's still useful this way, mirroring the public docs, so let's have it like this for the moment, even if it might bring some confusion down the line. This should be addressed as soon as we revamp our public docs. Thanks, resolving this!